examples/misc/upload_image_assets.py

```diff
@@ -55,13 +55,13 @@ def main(client, customer_id):
             asset_service.mutate_assets(customer_id,
                                         [asset_operation])
         )
-        print(f'Uploaded file(s):')
+        print('Uploaded file(s):')
         for row in mutate_asset_response.results:
             print(f'\tResource name: {row.resource_name}')
 
     except GoogleAdsException as ex:
-        print('Request with ID "%s" failed with status "%s" and includes the '
-              'following errors:' % (ex.request_id, ex.error.code().name))
+        print(f'Request with ID "{ex.request_id}" failed with status '
+              f'"{ex.error.code().name}" and includes the following errors:')
         for error in ex.failure.errors:
             print('\tError with message "%s".' % error.message)
```

Comment:
Sorry for not being more clear, when I mentioned consistently using f-strings, I also meant that for every following line.

Reply:
I think I got them all now at least in this file. There is a separate bug to track re: updating all the samples.